MODULE.bazel

```diff
@@ -45,7 +45,7 @@ bazel_dep(
 
 bazel_dep(
     name = "rules_go",
-    version = "0.40.1",
+    version = "0.41.0",
```

Comment:
We need this for https://github.com/bazelbuild/rules_go/commit/c8f0fd7735132144dcdf4d51e927a8607af6774c because https://github.com/bazelbuild/rules_go/issues/3644